documentation/style-guide.rst

```diff
@@ -102,6 +102,44 @@ Unix
    The name of the operating system developed at AT&T Bell Labs in the early
    1970s.
 
+Diátaxis
+========
+
+Python's documentation follows the `Diátaxis <https://diataxis.fr//>`_
+framework. This means adapting the writing style according to the nature of
+the documentation that is being written. The framework splits documentation
+into four distinct types: tutorials, how-to guides, reference, and
+explanation.
+
+`Python tutorials <https://docs.python.org/3/tutorial/index.html>`_ should be
+beginner-friendly, explicit, and avoid making assumptions about the reader's
+knowledge. The goal of a tutorial is to get the user writing Python code as
```

Comment:
In fact there is a single Python tutorial - this end-to-end learning journey that has been conceived and implemented, and I think that's an important fact about it. I think I would like to say something like:

> The goal of the tutorial is to provide a learning *experience*, in which a user is guided through a series of activities, and in doing so encounters concepts, actions tools and so on. The minimum possible explanation and technical detail should be provided (link to further information when required).

I'd actually say nothing about beginners - there could be in principle a very advanced tutorial.

Reply:
Also possible to link to https://diataxis.fr/tutorials/ - also available via InterSphinx.